tools/sdk-ai-bots/.gitignore

```diff
@@ -0,0 +1,122 @@
+# Logs
```

Comment:
Do we really need a specialized ignore file? Usually we just depend on the one in the root.